tests/compliance_operator_v2_test.go

```diff
@@ -0,0 +1,10 @@
+//go:build compliance
```

Comment:
Add build tags here so that these tests are only picked up when we invoke them with `make compliance-v2-tests`, and they aren't picked up when we run all API endpoint tests, which may not have the compliance operator installed.